compiler/backend_inkwell/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "backend_inkwell"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+candy_frontend = { version = "0.1.0", path = "../frontend" }
+inkwell = { version = "0.2.0", features = ["llvm15-0"] }
+llvm-sys = { version = "*", features = ["prefer-dynamic"] }
```

Version 150 corresponds to LLVM 15.0.x

```suggestion
llvm-sys = { version = "150", features = ["prefer-dynamic"] }
```